exchange/exchangetest/aliases.json

```diff
@@ -58,11 +58,7 @@
             }
           ],
           "ext": {
-            "prebid": {
-              "aliases": {
-                "districtm": "appnexus"
-              }
-            }
+            "prebid": {}
```

Comment:
There is an existing behavior where if `ext.prebid` is empty the empty object is written to the request. It's easy to remove this field instead. Is that safe? I imagine the current functionality is an accident, but it does provide a signal downstream that the request was processed by PBS. Do you think it's important to preserve that behavior? I'd personally love to make the request as compact/small as possible.